### PR TITLE
[uk]Added fan intents to Ukrainian language

### PR DESCRIPTION
**Pull request by skynetua**

### CODE REVIEW

File: tests/uk/_fixtures.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ language: uk
 areas:
 - name: Кухн(я|і)
   id: kitchen
-- name: Гостинн(а|ій)
+- name: Вітальн(я|і)
```
Comment (balloob) [inline tests/uk/_fixtures.yaml:5]: Names should not use matching syntax. They should be as you would name them in Home Assistant.

Comment (balloob) [reply tests/uk/_fixtures.yaml:5]: Why would you want to match on both?

Comment (skynetua) [reply tests/uk/_fixtures.yaml:5]: It is hard to explain in English, I’m not a linguist.

Basically in all Slavic languages we change the ending of nouns and adjectives depending on Cases. As you can see in `pl` situation is the same https://github.com/home-assistant/intents/blob/main/tests/pl/_fixtures.yaml

For example:
I want to name the kitchen area - it is ‘Кухня’
But if I want to reference something, like ‘Light at the kitchen’ - it is ‘Light на Кухні’

or: 
Lamp - Лампа
Turn on Lamp - Ввімкни Лампу

We change the last 1-2 letters in such cases or add vowels if the word ends with consonants.

I added it as is for now to not fail tests but there should be more complex solution.

Comment (balloob) [reply tests/uk/_fixtures.yaml:5]: Your name in Home Assistant is static. It does not contain sentence syntax. It should never have been allowed here. This needs to be solved differently. 

Cc @synesthesiam

Comment (nagyrobi) [reply tests/uk/_fixtures.yaml:5]: Hungarian similar:
Kitchen - Konyha
Lignt in the kitchen - Lámpa a konyhában
Turn on the light in the kitchen - Kapcsold fel a lámpát a konyhában

Comment (nagyrobi) [reply tests/uk/_fixtures.yaml:5]: And they also depend on other factors...

Comment (balloob) [reply tests/uk/_fixtures.yaml:5]: Check. Let's track it in #164. For now we're going to approve this PR as it's not new for UK.

**Pull request merged**